lamindb/models/artifact.py

```diff
@@ -1490,17 +1490,19 @@ def n_objects(self) -> int:
     def path(self) -> Path:
         """Path.
 
-        File in cloud storage, here AWS S3:
+        Example::
 
-        >>> artifact = ln.Artifact("s3://my-bucket/my-file.csv").save()
-        >>> artifact.path
-        S3QueryPath('s3://my-bucket/my-file.csv')
+            import lamindb as ln
 
-        File in local storage:
+            # File in cloud storage, here AWS S3:
+            artifact = ln.Artifact("s3://my-bucket/my-file.csv").save()
+            artifact.path
+            #S3QueryPath('s3://my-bucket/my-file.csv')
```

Comment:
When you want to indicate outputs, please use the convention from pydantic et al.

```
#> S3QueryPath('s3://my-bucket/my-file.csv') 
```